### PR TITLE
the clothing buckaroo

### PR DESCRIPTION
**Pull request by alsoandanswer**
the polaris PR 2 electric boogaloo

Basically deletes redundant sprites and replaces them with customizable and interchangeable versions.

Hoodies are still in. Leave me alone.

  - imageadd: "Fixes shading and missing inhands for high skirt. Adds skater skirt and tube skirt."
  - rscdel: "Removes some clothing from Polaris."
  - imageadd: "Beanies and some accessories have been resprited."
  - rscadd: "Adds cropped shirts and dress shirts, as well as a blouse."

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: code/modules/clothing/suits/hoodies.dm [hunk 1]
```diff
@@ -263,21 +263,3 @@
...
-	icon_state = "hoodie_crop"
-	icon_open = "hoodie_crop_open"
-	icon_closed = "hoodie_crop"
-	item_state = "hoodie_crop"
-	
-/obj/item/clothing/suit/storage/hooded/wintercoat/hoodie/sleeveless
-	icon_state = "hoodie_sleeveless"
-	icon_open = "hoodie_sleeveless_open"
-	icon_closed = "hoodie_sleeveless"
-	item_state = "hoodie_sleeveless"
```
Comment (TheStryker) [inline code/modules/clothing/suits/hoodies.dm:283]: Hoodies are gone here, but not in the loadout.

**Review by TheStryker — CHANGES_REQUESTED** [review]

File: code/modules/clothing/suits/hoodies.dm (hunk 1 above)
Comment (alsoandanswer) [reply code/modules/clothing/suits/hoodies.dm]: not anymore

**Review by TheStryker — APPROVED** [review]

File: html/changelogs/wezzy_polariskill.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - imageadd: "Fixes shading and missing inhands for high skirt. Adds skater skirt and tube skirt."
+  - rscdel: "Removes some clothing from Polaris."
```
Comment (NonQueueingMatt) [inline html/changelogs/wezzy_polariskill.yml:42]: tell people exactly what was removed

**Review by NonQueueingMatt — APPROVED** [review]

File: html/changelogs/wezzy_polariskill.yml (hunk 2 above)
Comment (alsoandanswer) [reply html/changelogs/wezzy_polariskill.yml:42]: aite

**Pull request merged**